Sources/SwiftCOM/Interfaces/Human/ID3D12CommandQueue.swift

```diff
@@ -46,9 +46,7 @@ public class ID3D12CommandQueue: ID3D12Pageable {
 
   public func GetDesc() throws -> D3D12_COMMAND_QUEUE_DESC {
     return try perform(as: WinSDK.ID3D12CommandQueue.self) { pThis in
-      var desc: D3D12_COMMAND_QUEUE_DESC = D3D12_COMMAND_QUEUE_DESC()
-      let _ = pThis.pointee.lpVtbl.pointee.GetDesc(pThis, &desc)
-      return desc
+      pThis.pointee.lpVtbl.pointee.GetDesc(pThis)
```

Comment:
This is almost 100% incorrect.  Do you have any documentation that indicates that this is a valid change?

The C signature for this is going to be `HRESULT GetDesc(ID3D12CommandQueue *, D3D12_COMMAND_QUEUE_DESC **)`.

Similar throughout.

Reply:
https://learn.microsoft.com/en-us/windows/win32/api/d3d12/nf-d3d12-id3d12commandqueue-getdesc

Reply:
That matches what I wrote above :).  The documentation is for C++ - the signature in C is different.